### PR TITLE
Swift 4 migration

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: JustPersist/Classes/Core/Implementations/ChildCoreDataStore.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 import CoreData
 
-@objc(JEChildCoreDataStore)
+@objcMembers
```
Comment (georgechauvet-je) [inline JustPersist/Classes/Core/Implementations/ChildCoreDataStore.swift:11]: Does this mean that we no longer have a 'JE' prefix when calling JustPersist from Objective-C code?

Comment (ghost) [reply JustPersist/Classes/Core/Implementations/ChildCoreDataStore.swift:11]: Correct

**Comment by georgechauvet-je** [comment]
When I load the example workspace, Xcode 9.1 tells me that conversion to Swift 4 is available. Does that mean that something is still missing?

**Comment by ghost** [comment]
@georgechauvet-je that's unexpected. What does it do if you perform the conversion?

**Comment by georgechauvet-je** [comment]
Apologies - I was looking at master by mistake - the swift 4 migration branch looks fine apart from a few recommendations that Xcode 9.1 suggests.

**Review by georgechauvet-je — APPROVED** [review]
LGTM

**Pull request merged**